recipes/bamhash/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
```

Comment:
Try this one: `CFLAGS=-I$PREFIX/include  make`
Moreover, make install does not exists. You need to copy the binaries to `$PREFIX/bin`

Have a look here for the name of binaries: https://github.com/bgruening/galaxytools/blob/master/tools/bamhash/tool_dependencies.xml